readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -1152,7 +1152,7 @@ def test_install_core_requirements_sphinx(self):
         requirements_sphinx = [
             'commonmark==0.5.4',
             'recommonmark==0.4.0',
-            'sphinx==1.7.5',
+            'sphinx<1.8',
```

Comment:
Feels like we shouldn't be checking for this in a test, as it makes it really brittle. Should we remove this test?

Reply:
Looking at it more, it looks like we should be checking for what we actually care about (eg. `'sphinx' in called_arguments` not "check it called the exact versions from our code"

Reply:
That makes sense, should I change all the tests here?

Reply:
Sure. I think it's a small enough change to be fine.